conversations.yaml

```diff
@@ -0,0 +1,536 @@
+          description: must be an E.164 format number.
+    OutboundMessageRequest:
+      properties:
+        recipient:
+          properties:
+            id:
+              type: string
+              description:
+                An identifier unique to the customer, such as phone number,
+                email, or another channel-specific value. For phone numbers,
+                the identifier must be in E.164 format.
+        message:
+          properties:
+            text:
+              type: string
+            quick_replies:
+              type: array
+              items:
+                properties:
+                  content_type:
+                    type: string
+                  title:
+                    type: string
+                  payload:
+                    type: string
+                  image_url:
+                    type: string
+                    format: uri
+            notification_type:
+              type: string
+              enum: [REGULAR]
+    OutboundMessageResponse:
+      properties:
+        recipient_id:
+          type: string
+          description:
+            The identifier for the recipient, as sent in the original
+            request.
+        message_id:
+          type: string
+          description:
+            A unique identifier for the message, assigned by the
+            channel adapter.
+    ForwardedMessage:
+      properties:
+        id:
+          type: string
+          description:
+            The PageId value configured in Conversations at the creation of the channel.
+        time:
+          type: integer
+          format: int64
+          description:
+            Unix epoch timestamp of the last event associated with the
+            request. For example, when it is a multi-part message, this
+            is the time of the last message-part.
+        messaging:
+          type: array
+          items:
+            properties:
+              sender:
+                properties:
+                  id:
+                    type: string
+                    description:
+                      An identifier unique to the customer, such as phone
+                      number, email, or other channel-specific value.
+                      For phone numbers, the ID must be in E.164 format.
+              recipient:
+                properties:
+                  id:
+                    type: string
+                    description:
+                      The PageId value configured in Conversations at the creation of the channel..
+              timestamp:
+                type: integer
+                format: int64
+                description:
+                  Unix epoch timestamp of the last event associated with the
+                  request. For example, when it is a multi-part message, this
+                  is the time of the last message-part.
+              message:
+                properties:
+                  mid:
+                    type: string
```

Comment:
What is the `mid`?  Can you add a `description:` to explain and any relevant `format:` (if applicable)?